telluric/georaster.py

```diff
@@ -1561,66 +1520,31 @@ def get_window(self, window, bands=None,
         :param masked: boolean, if `True` the return value will be a masked array. Default is True
         :return: GeoRaster2 of tile
         """
-        xratio, yratio = self._get_widow_calculate_resize_ratio(xsize, ysize, window)
         bands = bands or list(range(1, self.num_bands + 1))
-        out_shape = self._get_window_out_shape(bands, xratio, yratio, window)
-        # if window and raster dont intersect return an empty raster in the requested size
-        if not self._window_intersects_with_raster(window):
-            array = np.zeros(out_shape, dtype=self._dtype)
-            affine = self._calculate_new_affine(window, out_shape[2], out_shape[1])
-            return self.copy_with(image=array, affine=affine)
-
-        requested_window = self._get_window_requested_window(window, boundless)
 
         # requested_out_shape and out_shape are different for out of bounds window
-        requested_out_shape = self._get_window_out_shape(bands, xratio, yratio, requested_window)
+        out_shape = self._get_window_out_shape(bands, window, xsize, ysize)
         try:
             read_params = {
-                "window": requested_window,
+                "window": window,
                 "resampling": resampling,
-                "boundless": boundless,
+                "boundless": True,
                 "masked": masked,
-                "out_shape": requested_out_shape
+                "out_shape": out_shape
             }
 
-            rasterio_env = {
```

Comment:
I guess this is part of

>  Therefore I decided to remove the boundless code completely

Reply:
Well it is part of what we dicussed yesterday, read should work on the generic case, 
if you want something special you will need to set in the env variables your self